src/main/java/com/squareup/javapoet/MethodSpec.java

```diff
@@ -292,11 +292,16 @@ public Builder toBuilder() {
     private CodeBlock defaultValue;
 
     private Builder(String name) {
+      setName(name);
+    }
+
+    public Builder setName(String name) {
       checkNotNull(name, "name == null");
       checkArgument(name.equals(CONSTRUCTOR) || SourceVersion.isName(name),
-          "not a valid name: %s", name);
+              "not a valid name: %s", name);
```

Comment:
Could you please revert this?

Reply:
Oops. Fixed fe3f339c168ddca267c2e6ae0aad095b455d8c5f